be1-go/channel/coin/uint53/mod_test.go

```diff
@@ -0,0 +1,35 @@
+package uint53
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_Uint53Max_Equivalent(t *testing.T) {
+	require.Equal(t, Uint53(1<<53-1), MaxUint53)
+}
+
+func Test_InRange_Examples(t *testing.T) {
+	require.True(t, InRange(0))
+	require.True(t, InRange(MaxUint53))
+	require.False(t, InRange(MaxUint53+1))
+}
+
+func Test_SafePlus_Examples(t *testing.T) {
```

Comment:
you should also add cases where an error is expected: 

```go
SafePlus(MaxUint53, 1)
SafePlus(MaxUint53+1, 0)
```

Reply:
`SafePlus(MaxUint53, 1)` is already a test case, `SafePlus(MaxUint53+1, 0)` violates pre-conditions.

Reply:
added at the bottom